<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="HttpException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac51864e8fe52f0eb87b7c23fbfe505d15cf9681ede.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1864e8fe52f0eb87b7c23fbfe505d15cf9681ede</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Describes an exception that occurred during the processing of HTTP requests.</source>
          <target state="translated">Descrive un'eccezione che si è verificata durante l'elaborazione delle richieste HTTP.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class is an HTTP-specific exception class that enables ASP.NET to generate exception information.</source>
          <target state="translated">La <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> classe è una classe di eccezione specifica di HTTP che consente ad ASP.NET di generare le informazioni sull'eccezione.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information about throwing and handling exceptions, see <bpt id="p1">[</bpt>Exceptions<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</source>
          <target state="translated">Per ulteriori informazioni sulla generazione e la gestione delle eccezioni, vedere <bpt id="p1">[</bpt>eccezioni<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The following code example demonstrates how to use the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class to raise customized exceptions.</source>
          <target state="translated">Esempio di codice seguente viene illustrato come utilizzare la <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> classe per la generazione di eccezioni personalizzate.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">`Button_Click`</ph> method determines which button was clicked, and then calls either the <ph id="ph2">`CheckNumber`</ph> or <ph id="ph3">`CheckBoolean`</ph> method.</source>
          <target state="translated">Il <ph id="ph1">`Button_Click`</ph> metodo determina quale pulsante è stato scelto e quindi chiama il <ph id="ph2">`CheckNumber`</ph> o <ph id="ph3">`CheckBoolean`</ph> metodo.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>If the user-entered value in the text box does not correspond to the expected type, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is created and thrown with a customized message based on the error type.</source>
          <target state="translated">Se il valore immesso dall'utente nella casella di testo non corrisponde al tipo previsto, un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> eccezione viene creata e generata un messaggio personalizzato in base al tipo di errore.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class and creates an empty <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> object.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> e crea un oggetto <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> vuoto.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Quando la gestione delle eccezioni, può essere utile acquisire una serie di eccezioni correlate con l'eccezione esterna generata in risposta all'eccezione interna.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Un riferimento all'eccezione interna che ha causato l'eccezione esterna è disponibile il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> proprietà dell'eccezione esterna.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Questo meccanismo consente di mantenere le informazioni sull'errore che viene eseguite da eccezioni precedenti, incluse le eccezioni originali, consentendo di creare eccezioni esterne più significative.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Per ulteriori informazioni, vedere <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated">Il <ph id="ph1">`CheckNumber`</ph> metodo accetta un valore immesso dall'utente mediante una casella di testo e verifica se si tratta di un numero intero.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Se il valore non è un numero intero, viene generata un'eccezione e quindi un nuovo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> oggetto viene creato e generato.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">L'eccezione viene rilevata nel <ph id="ph1">`Button_Click`</ph> gestore dell'evento e il messaggio di errore viene visualizzato nel browser.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using a supplied error message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un messaggio di errore specificato.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is thrown if a user-entered value is 0.</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> viene generata se un valore immesso dall'utente è 0.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The HTTP response status code sent to the client corresponding to this error.</source>
          <target state="translated">Codice di stato della risposta HTTP inviato al client corrispondente all'errore.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code and an error message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un codice di stato della risposta HTTP e un messaggio di errore.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> che contiene i dati oggetto serializzati relativi all'eccezione generata.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated">Elemento <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> che contiene le informazioni contestuali sull'origine o sulla destinazione.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> con dati serializzati.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore viene chiamato durante la deserializzazione per ricostituire l'oggetto eccezione trasmesso in un flusso.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">Per ulteriori informazioni, vedere <bpt id="p1">[</bpt>serializzazione XML e SOAP<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">Elemento <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, se presente, che ha generato l'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un messaggio di errore e la proprietà <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Quando la gestione delle eccezioni, può essere utile acquisire una serie di eccezioni correlate con l'eccezione esterna generata in risposta all'eccezione interna.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Un riferimento all'eccezione interna che ha causato l'eccezione esterna è disponibile il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> proprietà dell'eccezione esterna.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Questo meccanismo consente di mantenere le informazioni sull'errore che viene eseguite da eccezioni precedenti, incluse le eccezioni originali, consentendo di creare eccezioni esterne più significative.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Per ulteriori informazioni, vedere <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated">Il <ph id="ph1">`CheckNumber`</ph> metodo accetta un valore immesso dall'utente mediante una casella di testo e verifica se si tratta di un numero intero.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then in the catch block, a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Se il valore non è un numero intero, viene generata un'eccezione e quindi, nel blocco catch, un nuovo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> oggetto viene creato e generato.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">L'eccezione viene rilevata nel <ph id="ph1">`Button_Click`</ph> gestore dell'evento e il messaggio di errore viene visualizzato nel browser.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">Codice di eccezione che definisce l'errore.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and an exception code.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un messaggio di errore e un codice di eccezione.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is thrown if a user-entered value is 0.</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> eccezione viene generata se un valore immesso dall'utente è 0.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">Codice di stato della risposta HTTP visualizzato sul client.</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">Elemento <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, se presente, che ha generato l'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un codice di stato della risposta HTTP, un messaggio di errore e la proprietà <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Quando la gestione delle eccezioni, può essere utile acquisire una serie di eccezioni correlate con l'eccezione esterna generata in risposta all'eccezione interna.</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Un riferimento all'eccezione interna che ha causato l'eccezione esterna è disponibile il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> proprietà dell'eccezione esterna.</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Questo meccanismo consente di mantenere le informazioni sull'errore che viene eseguite da eccezioni precedenti, incluse le eccezioni originali, consentendo di creare eccezioni esterne più significative.</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Per ulteriori informazioni, vedere <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value and checks whether it is an integer.</source>
          <target state="translated">Il <ph id="ph1">`CheckNumber`</ph> metodo accetta un valore immesso dall'utente e verifica se si tratta di un numero intero.</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object containing the HTTP response status code, the exception's message, and any inner exception is created.</source>
          <target state="translated">Se il valore non è un numero intero, viene generata un'eccezione e quindi un nuovo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> viene creato l'oggetto che contiene il codice di stato di risposta HTTP, il messaggio dell'eccezione e qualsiasi eccezione interna.</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message, error code, and inner exception are displayed.</source>
          <target state="translated">L'eccezione viene rilevata nel <ph id="ph1">`Button_Click`</ph> vengono visualizzati gestore dell'evento e il messaggio di errore, il codice di errore e l'eccezione interna.</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="174" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="175" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">Codice di stato della risposta HTTP visualizzato sul client.</target>       </trans-unit>
        <trans-unit id="176" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="177" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">Codice di eccezione che definisce l'errore.</target>       </trans-unit>
        <trans-unit id="178" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and an exception code.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> usando un codice di stato della risposta HTTP, un messaggio di errore e un codice di eccezione.</target>       </trans-unit>
        <trans-unit id="179" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">L'esempio di codice seguente illustra il <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> costruttore del <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="180" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The user name and email information are entered by the user in the provided text boxes.</source>
          <target state="translated">Le informazioni utente, nome e alla posta elettronica vengono immessi dall'utente nelle caselle di testo fornito.</target>       </trans-unit>
        <trans-unit id="181" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>If any of the text boxes are left blank, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Se una delle caselle di testo sono vuoti, un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> oggetto viene creato e generato.</target>       </trans-unit>
        <trans-unit id="182" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error code of the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is obtained by the <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> method and displayed on the Web page.</source>
          <target state="translated">Il codice di errore di <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> consente di ottenere il <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> (metodo) e visualizzato nella pagina Web.</target>       </trans-unit>
        <trans-unit id="183" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">L'esempio include una casella di testo che accetta l'input dell'utente e rappresenta quindi una potenziale minaccia alla sicurezza.</target>       </trans-unit>
        <trans-unit id="184" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">Per impostazione predefinita, le pagine Web ASP.NET verificano che l'input dell'utente non includa script o elementi HTML.</target>       </trans-unit>
        <trans-unit id="185" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Per altre informazioni, vedere <bpt id="p1">[</bpt>Cenni preliminari sugli attacchi tramite script<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="186" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Messaggio di errore visualizzato dal client quando viene generata l'eccezione.</target>       </trans-unit>
        <trans-unit id="187" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>Creates a new <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> exception based on the error code that is returned from the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated">Crea una nuova eccezione <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> basata sul codice di errore restituito dal metodo <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> API Win32.</target>       </trans-unit>
        <trans-unit id="188" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>An <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> based on the error code that is returned from a call to the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated">Elemento <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> basato sul codice di errore restituito da una chiamata al metodo <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> API Win32.</target>       </trans-unit>
        <trans-unit id="189" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>Gets the HTML error message to return to the client.</source>
          <target state="translated">Ottiene il messaggio di errore HTML da restituire al client.</target>       </trans-unit>
        <trans-unit id="190" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>The HTML error message.</source>
          <target state="translated">Messaggio di errore HTML.</target>       </trans-unit>
        <trans-unit id="191" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Gets the HTTP response status code to return to the client.</source>
          <target state="translated">Ottiene il codice di stato della risposta HTTP da restituire al client.</target>       </trans-unit>
        <trans-unit id="192" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>A non-zero HTTP code representing the exception or the <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> code; otherwise, HTTP response status code 500.</source>
          <target state="translated">Codice HTTP diverso da zero che rappresenta l'eccezione o codice <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>; in caso contrario, stato della risposta HTTP 500 del codice.</target>       </trans-unit>
        <trans-unit id="193" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If there is a non-zero HTTP code, it is returned.</source>
          <target state="translated">Se è presente un codice HTTP diverso da zero, viene restituito.</target>       </trans-unit>
        <trans-unit id="194" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Otherwise, the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code is returned.</source>
          <target state="translated">In caso contrario, il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> codice viene restituito.</target>       </trans-unit>
        <trans-unit id="195" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If neither an <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code nor a non-zero HTTP code is available, the HTTP response status code 500 is returned.</source>
          <target state="translated">Se non si specifica un <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> è disponibile codice né un codice HTTP diverso da zero, viene restituito il codice di stato di risposta HTTP 500.</target>       </trans-unit>
        <trans-unit id="196" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>For more information on HTTP response status codes, see the Status Code Definitions of the Hypertext Transfer Protocol - HTTP/1.1 on the World <bpt id="p1">[</bpt>Wide Web Consortium (W3C) Web site<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>.</source>
          <target state="translated">Per ulteriori informazioni sui codici di stato di risposta HTTP, vedere le definizioni dei codici di stato di Hypertext Transfer Protocol - HTTP/1.1 nel mondo <bpt id="p1">[</bpt>sito Web Wide Web Consortium (W3C)<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>.</target>       </trans-unit>
        <trans-unit id="197" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> che contiene i dati oggetto serializzati relativi all'eccezione generata.</target>       </trans-unit>
        <trans-unit id="198" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated">Elemento <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> che contiene le informazioni contestuali sull'origine o sulla destinazione.</target>       </trans-unit>
        <trans-unit id="199" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Gets information about the exception and adds it to the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object.</source>
          <target state="translated">Ottiene informazioni sull'eccezione e le aggiunge all'oggetto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="200" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> method adds information about the exception to the <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object that is targeted for serialization.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> metodo aggiunge informazioni sull'eccezione per il <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> oggetto di destinazione per la serializzazione.</target>       </trans-unit>
        <trans-unit id="201" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> that is transmitted over the stream.</source>
          <target state="translated">Durante la deserializzazione, l'eccezione viene ricostituita dal <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> che viene trasmesso sul flusso.</target>       </trans-unit>
        <trans-unit id="202" translate="yes" xml:space="preserve" uid="P:System.Web.HttpException.WebEventCode">
          <source>Gets the event codes that are associated with the HTTP exception.</source>
          <target state="translated">Ottiene i codici di evento associati all'eccezione HTTP.</target>       </trans-unit>
        <trans-unit id="203" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpException.WebEventCode">
          <source>An integer representing a Web event code.</source>
          <target state="translated">Integer che rappresenta il codice di un evento Web.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>